packages/block-library/src/navigation-link/index.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * Server-side rendering of the `core/navigation-link` block.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Returns the top-level submenu SVG chevron icon.
+ *
+ * @return string
+ */
+function block_core_navigation_link_render_submenu_icon() {
+	return '<svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" viewBox="0 0 24 24" transform="rotate(90)"><path d="M8 5v14l11-7z"/><path d="M0 0h24v24H0z" fill="none"/></svg>';
+}
+
+/**
+ * Renders the `core/navigation-link` block.
+ *
+ * @param array $content The saved content.
+ * @param array $block   The parsed block.
+ *
+ * @return string Returns the post content with the legacy widget added.
+ */
+function render_block_core_navigation_link( $content, $block ) {
+	if ( 'core/navigation-link' !== $block['blockName'] ) {
+		return $content;
+	}
+
+	// Apply default attributes. This is currently needed because the nav link block overrides
+	// block rendering using the render_block filter. It does this to gain access to
+	// innerBlocks, but at the same time this bypasses prepare_attributes_for_render.
```

Comment:
Is there no other way to access inner blocks? I remember @draganescu running into this—what did we end up doing? (I can't find the issue!)

Reply:
What we ended up doing to access innerBlocks, using the `render_block` filter is the problem why this comment exists.

Reply:
Got it 👍